controls/RHEL-07-021010.rb

```diff
@@ -35,8 +35,13 @@
 If a file system found in “/etc/fstab” refers to the user home directory file system and it does not have the “nosetuid” option set, this is a finding.'
 
 # START_DESCRIBE RHEL-07-021010
-  describe file('') do
-    it { should match // }
+  fstab_lines = file('/etc/fstab').content.split("\n")
```

Comment:
What this line is trying to do is read the fstab file and convert new lines into a list. Because the list content of the file Null/None for some reason (Does the file exist?, Do you have permission to read it?) it won't convert it to a list. Then it tries to loop the lines and match lines have "home" in it. If it does exist it creates a control. My only solution/idea I can come up with now is to catch the error but you will still see ``No tests executed``. I'm fighting over this one, it would be nice if there was mechanism to have the control exist and only fail if there is a specific line in the file.